src/Data/Rational/Properties.agda

```diff
@@ -1331,26 +1331,26 @@ module _ where
 *-cancelˡ-≤-pos : ∀ r .{{_ : Positive r}} → r * p ≤ r * q → p ≤ q
 *-cancelˡ-≤-pos {p} {q} r rewrite *-comm r p | *-comm r q = *-cancelʳ-≤-pos r
 
-*-monoʳ-≤-nonNeg : ∀ r .{{_ : NonNegative r}} → (_* r) Preserves _≤_ ⟶ _≤_
+*-monoʳ-≤-nonNeg : ∀ r .{{_ : NonNegative r}} → Monotonic₁ _≤_ _≤_ (_* r)
```

Comment:
Feels like the 8 lemmas below here should use `Left`/`RightMonotonic`? Maybe same in `Data.Integer.Properties`?

Reply:
This is why I'm currently stalling on the reviews of #2573 , because this PR will then itself need the analogues of `_-AlmostLeftMonotonic_` etc., and I didn't quite have the energy to add them... yet.

Reply:
Oh right of course. Ignore this comment. Best to get this merged in if we can.